electrumsv/wallet.py

```diff
@@ -104,6 +105,28 @@ def key(self):
     def key_str(self):
         return ':'.join((self.tx_hash, str(self.out_index)))
 
+    def as_dict(self):
```

Comment:
I think this should be removed and moved out to the example application for it's own purposes.